app/src/main/java/com/github/steroidteam/todolist/todo/Task.java

```diff
@@ -5,13 +5,13 @@
 /**
  * A class that represents a task described by a title and a body.
  * <p>
- * The id is null upon the creation of the Task, and it will
+ * The index is null upon the creation of the Task, and it will
  * be assigned by the database when it's pushed into it.
  */
-public class Task implements Serializable {
+public class Task {
 
     private String body;
-    private Integer id;
+    private Integer index;
 
```

Comment:
Wouldn't it make sense to also add a boolean isChecked (or isDone) attribute, in order to remember which tasks the user has checked ?

Reply:
Sure, I will add it.